lib/v2/oshwhub/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'oshwhub.com': {
+        _name: '立创开源硬件平台',
+        '.': [
+            {
+                title: '开源广场',
+                docs: 'https://docs.rsshub.app/other.html#li-chuang-kai-yuan-ying-jian-ping-tai',
+                source: ['/explore', '/explore?projectSort=:sortType'],
+                target: '/oshwhub/:sortType',
```

Comment:
You can't match URL Search Parameters in `source`
You need to use `target` function
docs: https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-radar-gui-ze-bian-xie-gui-ze-source